perk-store-services/src/main/java/org/exoplatform/perkstore/storage/PerkStoreStorage.java

```diff
@@ -156,11 +156,11 @@ public double countUserPurchasedQuantityInPeriod(long productId, long identityId
     }
   }
 
-  public List<ProductOrder> getOrders(String username, OrderFilter filter) {
+  public List<ProductOrder> getOrders(String username, OrderFilter filter, Boolean isPerkStoreManager) {
```

Comment:
```suggestion
  public List<ProductOrder> getOrders(String username, OrderFilter filter, boolean isPerkStoreManager) {
```
If we don't use null value, it's better to use primitive value in method signature